### DISCUSSION TITLE
Report Function with new API

### DISCUSSION THREAD

**Discussion by Anjerlaan** [Koenkk/zigbee2mqtt#5508] [Devices]
@Koenkk, I noticed that in version 1.17.0 the "Report" function can be turned on per device in the webbased frontend.
In a previous release I have turned on the report function in the configuration.yaml.
The only reason I did this was for the IKEA Remotes I use.

Does the new feature mean that this can be turned off in the configuration.yaml.....and turn on report per (remote) device in the frontend?

Is there also a "help" file of the options (which endpoint, clusters and attributes to select for e.g. IKEA remotes)?

**Comment by Koenkk** [comment]
The report feature will become deprecated, reporting will be automatically configured when you bind devices (but I have not implemented this yet). You can indeed disable the report feature and configure the reportings yourself but I suggest to wait when this new functionality is in place.

**Reply by Anjerlaan** [reply]
> The report feature will become deprecated, reporting will be automatically configured when you bind devices (but I have not implemented this yet). You can indeed disable the report feature and configure the reportings yourself but I suggest to wait when this new functionality is in place.

When this function is in place, will it also automatically configure reporting to already bounded devices or do we need to rebind them?

**Reply by Koenkk** [reply]
This depends wether the binds were saved (this has been added some time ago). You can check this via the frontend by clicking on the device and going to the bind tab.
